docker/jupyterlab/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Use tensorflow-1.14.0 as a base image, allowing the user to
+# speficy if they want GPU support, by setting IMAGE_TYPE to "gpu".
+ARG IMAGE_TYPE="cpu"
+FROM gcr.io/kubeflow-images-public/tensorflow-1.14.0-notebook-${IMAGE_TYPE}:v0.7.0
+
+USER root
+
+# Install basic dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+        ca-certificates bash-completion tar less \
+        python-pip python-setuptools build-essential python-dev \
+        python3-pip python3-wheel && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV SHELL /bin/bash
+COPY bashrc /etc/bash.bashrc
+RUN echo "set background=dark" >> /etc/vim/vimrc.local
+
```

Comment:
Why don't we run a `pip install jupyterlab` like in the KubeCon based Dockerfiles? I propose having `pip install 'jupyterlab<2.0.0'` here and in `Dockefile.dev`as well.

Reply:
I meant to remove it from the `.rok` ones as well, but forgot it.

The idea was to use the version installed in official KF images which I've used as base. But I'll fix the dockerfiles as your comments suggest, to show that `jupyterlab` has to be installed in case someone decides to use a different base.